src/main/java/engineering/swat/watch/impl/jdk/JDKRecursiveDirectoryWatch.java

```diff
@@ -48,6 +48,7 @@
 import org.apache.logging.log4j.Logger;
 
 import engineering.swat.watch.WatchEvent;
+import engineering.swat.watch.WatchScope;
 import engineering.swat.watch.impl.EventHandlingWatch;
 
 public class JDKRecursiveDirectoryWatch extends JDKBaseWatch {
```

Comment:
I expected a lot more changes in this file? as it's currently also generating events while it's iterating the directories?